v21.1/connect-to-the-database-cockroachcloud.md

```diff
@@ -34,7 +34,13 @@ Do the following:
 
 <section class="filter-content" markdown="1" data-scope="sql">
 
-{% remote_include https://raw.githubusercontent.com/cockroachdb/docs/master/_includes/cockroachcloud/connect_to_crc.md|<!-- BEGIN CRC free sql -->|<!-- END CRC free sql --> %}
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql \
+--url='postgres://<username>:<password>@<global host>:26257/<database>?sslmode=verify-full&sslrootcert=<path to the CA certificate>'
+~~~
+
+{% include {{page.version.version}}/app/cc-free-tier-params.md %}
```

Comment:
IDK how much this matters, but the parameters in this include file use `{`/`}`, and the example uses `<`/`>` for the variable values. It would look nicer if they used the same format.

The same is true for a couple other examples on this page.